packages/terra-clinical-onset-picker/src/OnsetUtils.js

```diff
@@ -130,6 +120,38 @@ class OnsetUtils {
     return { age: ageDiff, ageUnit: 'weeks' };
   }
 
+  /**
+   * Converts onset date to a age value with lowest possible age unit (weeks, then months, then years).
```

Comment:
I'd think that we could stick to having one utility method instead of two - description of `onsetToAge` is the same as that of `initialOnsetToAge` and does not tell when either of them should be leveraged.

Reply:
Removed the previous one 1417f94